pdsview/pdsview.py

```diff
@@ -2,29 +2,147 @@
 
 import sys
 import os
-# import logging
-import PDSImage
-import label
-
+import logging
+try:
+    import label
+except:
+    from pdsview import label
+from glob import glob
 from ginga.qtw.QtHelp import QtGui, QtCore
 from ginga.qtw.ImageViewCanvasQt import ImageViewCanvas
+from ginga.BaseImage import BaseImage
+from planetaryimage import PDS3Image
+import argparse
 
 STD_FORMAT = '%(asctime)s | %(levelname)1.1s | %(filename)s:%(lineno)d (%(funcName)s) | %(message)s'
 #
 #
-# class PDSModel():
-#
+app = QtGui.QApplication.instance()
```

Comment:
It remember at one point it wasn't global but I couldn't test certain things. When I made it global, testing became easier. I also think it allows you to rerun pdsview from a python interpreter